src/app/component/auth/new-sign-up/new-sign-up.component.ts

```diff
@@ -9,13 +18,135 @@ import { MatDialogRef } from '@angular/material/dialog';
 })
 export class NewSignUpComponent implements OnInit {
   private signUpImgs = authImages;
+  private userOwnSignUp: UserOwnSignUp;
+  private firstNameErrorMessageBackEnd: string;
+  private lastNameErrorMessageBackEnd: string;
+  private emailErrorMessageBackEnd: string;
+  private passwordErrorMessageBackEnd: string;
+  private passwordConfirmErrorMessageBackEnd: string;
+  private loadingAnim = false;
+  private tmp: string;
+  private backEndError: string;
 
-  constructor(private matDialogRef: MatDialogRef<NewSignUpComponent>) { }
+  constructor(private matDialogRef: MatDialogRef<NewSignUpComponent>,
+              private dialog: MatDialog,
+              private userOwnSignInService: UserOwnSignInService,
+              private userOwnSecurityService: UserOwnSignUpService,
+              private router: Router,
+              private authService: AuthService,
+              private googleService: GoogleSignInService) { }
 
   ngOnInit() {
+    this.userOwnSignUp = new UserOwnSignUp();
+    this.setNullAllMessage();
+  }
+
+  private setNullAllMessage(): void {
+    this.firstNameErrorMessageBackEnd = null;
+    this.lastNameErrorMessageBackEnd = null;
+    this.emailErrorMessageBackEnd = null;
+    this.passwordErrorMessageBackEnd = null;
+    this.passwordConfirmErrorMessageBackEnd = null;
+  }
+
+  private onSubmit(userOwnRegister: UserOwnSignUp): void {
+    this.setNullAllMessage();
+    this.loadingAnim = true;
+    this.userOwnSecurityService.signUp(userOwnRegister).subscribe(
+      () => {
```

Comment:
Please rewrite it(subscribe(this.onSubmitSuccess.bind(this), ...)

Reply:
Fixed